package.json

```diff
@@ -4,9 +4,10 @@
   "description": "TypeScript loader for webpack",
   "main": "index.js",
   "scripts": {
-    "pretest": "node build",
+    "build": "node scripts/build",
+    "pretest": "npm run build",
     "test": "node ./node_modules/mocha/bin/mocha --reporter spec test/run.js",
```

Comment:
I haven't really given much thought to code coverage, but :+1: if you want to implement it :D